### DISCUSSION TITLE
Tiny UI on Steam Deck

### DISCUSSION THREAD

**Discussion by woj-tek** [Heroic-Games-Launcher/HeroicGamesLauncher#1287] [General]
I followed the guide (https://www.youtube.com/watch?v=8Zrz9WMbqP4) to install Heroic on SteamDeck and it installed without issue but I have an issue that the UI is not scaled properly (looks like it's 2x smaller and touchpad input operates only on 50% central part when launched from within SteamUI). When I run it in desktop mode it's scaled properly and I can navigate it using deck controllers and touch screen. How can I resolve it and run it nicely from within SteamUI?

(I hope it's correct place to ask this sort of questions)

EDIT: and of course after asking this question it started working ok....

**Comment by iohanr** [comment]
I have the same problem... just installed it today.  Did you do anything to get it working properly?

**Comment by woj-tek** [comment]
Hmm, I think my problem was caused by installing and starting Heroic with external monitor attached. After disconnecting it and restarting Heroic (and I think SteamDeck) it was scaled properly.

**Reply by iohanr** [reply]
I did install it with external monitor/kb/mouse attached. However, I have since disconnected and restarted heroic and rebooted the steam deck multiple times and still have the same issue.  Maybe I should uninstall and try it again from scratch but this time without external monitor.

**Reply by woj-tek** [reply]
I would do that. As I said in my case it "fixed itself" (after I asked the question), though I don't have SD handy.

**Reply by iohanr** [reply]
that did it!  uninstall/reinstall with no external monitor worked.  thanks!

**Comment by felipecrs** [comment]
Just to summarize, as this is still an issue to date (Heroic 2.15.1), the fix is to enter Discover, Remove Heroic and Install it again. This will not delete any data nor the saved credentials.

**Comment by jovanemarques** [comment]
For future reference. My screen was showing big instead of tiny. The reinstall trick didn't work here, so I did the following:

I changed directly in the config file and it was fixed. File is:

~/var/app/com.heroicgameslauncher.hgl/config/heroic/store/config.json

Change "zoomPercent" to the value you need. In my case it was huge (200) I changed to 100 and now is perfect.
